README.md

```diff
@@ -93,63 +112,172 @@ const strapiConfig = {
+    }
+  }
+}
+```
+
+#### Rich text field
+
+Rich text fields can now be processed using the [`gatsby-transformer-remark`](https://www.gatsbyjs.com/plugins/gatsby-transformer-remark/https://www.gatsbyjs.com/plugins/gatsby-transformer-remark/) plugin.
+
+> It only works if the content of the rich text field saved in the database is in markdown format so if you customised the WYSIWYG in your Strapi Administration panel make sure that it is saved in a markdown format.
+
+Files that are added in the richtext field can now also be processed by the [`gatsby-plugin-image`](https://www.gatsbyjs.com/plugins/gatsby-plugin-image/?=gatsby-plugin-i#gatsby-plugin-image) plugin.
+
+> To do so, according the [restrictons and limitations of the plugin](#restrictions-and-limitations) you need to make sure that at least one of your content types entities has a file uploaded in the richtext field.
+
+To query markdown local fields use the following query:
+
+```graphql
+{
+  allStrapiArticle {
+    nodes {
+      # richtext field content
+      body {
+        # object to access the markdown node
         data {
+          # unprocessed data from Strapi
+          body
+          # processed markdown
           childMarkdownRemark {
             html
+            rawMarkdownBody
           }
         }
-        # Extracted files from the richtext field
+        # files from the markdown that are processed using the gatsby-image-plugin
         medias {
+          # alternative text saved in the markdown
+          alternativeText
+          # file from the media library
+          file {
+            # alternative text of the file in the media library
+            # it can be different from the one set in your markdown
+            alternativeText
+          }
+          # file processed with gatsby-plugin-image
           localFile {
             childImageSharp {
-              gatsbyImageData
+              gatsbyImageData(formats: NO_CHANGE)
             }
           }
-          alternativeText
-          # Original url in the markdown
+          # src set in your markdown field (ex: [alternativeText](src))
           src
-          # Prefixed url
+          # prefixed url with the Strapi api endpoint of the file
+          # when using a provider the src field value is equal to the url field value
           url
-          # Original media from the media library
-          file
+        }
+      }
+    }
+  }
+}
+```
+
+<!-- #### Internationalization support
+
+Strapi now supports [internationalization](https://strapi.io/documentation/developer-docs/latest/development/plugins/i18n.html#installation). But by default, this plugin will only fetch data in the default locale of your Strapi app. If your content types are available in different locales, you can also pass an entity definition object to specify the locale you want to fetch for a content type. Use the `all` value to get all available locales on a collection type. -->
+
+#### Components
```

Comment:
I think it's cool that this is possible, but I'm not sure we should encourage it. I'm afraid some people may start using components instead of content types and get confused.

Also, I think we should at least mention that this is a Gatsby id and not a Strapi one

Reply:
I think it's the correct way to use components in Gatsby therefore it's nice to document them, people cannot use components over content types as they cannot create components without content types...